docs/ref/forms/renderers.txt

```diff
@@ -114,6 +127,19 @@ Some widgets add further information to the context. For instance, all widgets
 that subclass ``Input`` defines ``widget['type']`` and :class:`.MultiWidget`
 defines ``widget['subwidgets']`` for looping purposes.
 
+Overriding built-in form templates
+==================================
+
+.. versionadded:: 4.0
+
+.. _overriding-built-in-form-templates:
+
+:attr:`.Form.template_name`
+
+To override form templates, you must use the :class:`TemplatesSetting`
+renderer. Then overriding widget templates works :doc:`the same as
```

Comment:
This is unclear; I'd expect to see a listing of the relevant template names here